spec/api/v2/api_authorization_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "spec_helper"
+require "webmock"
+require "api/api_helper"
+
+describe "API V2 Authorization" do
+  include ApiHelper
+  end_points = %w(/api/v2/works /api/v2/bookmarks)
+
+  describe "API POST with invalid request" do
+    it "should return 401 Unauthorized if no token is supplied and forgery protection is enabled" do
```

Comment:
I won't pester you to change the old `v1` tests, but if the new tests could be rephrased without the `should`, that would be great!